source/sass/views/home.scss

```diff
@@ -114,3 +114,64 @@
     }
   }
 }
+
+/**
+* Partner section
+*/
+.partner-section {
+  background-size: cover;
+  background-position: center center;
+  position: relative;
+  z-index: 10;
+  padding-top: 80px;
+  padding-bottom: 80px;
+}
+
+.partner-section::before {
+  content: '';
+  position: absolute;
+  z-index: 0;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.6);
+}
+
+.partner-section-text {
+  font-family: $font-family-sans-serif;
+  font-size: 20px;
+  font-weight: 300;
+  line-height: 1.5;
+  max-width: 450px;
```

Comment:
From the comps it seems like we want the text section above logs to take up certain columns on desktop/tablet/mobile instead of having a max width set to `450px` for all screen sizes. @taisdesouzalessa can you confirm?

Reply:
Marking as resolved as this PR passed design review already.